src/include/daos/tests_lib.h

```diff
@@ -36,26 +36,21 @@
 #define DAOS_ON_VALGRIND 0
 #endif
 
-#define assert_success(r) do {					              \
-	int __rc = (r);							      \
-	if (__rc != 0)							      \
-		fail_msg("Not successful!! Error code: " DF_RC, DP_RC(__rc)); \
+#define assert_success(r)						\
+	do {								\
+		int __rc = (r);						\
+		if (__rc != 0)						\
+			fail_msg("Not successful!! Error code: "	\
+				DF_RC, DP_RC(__rc));			\
```

Comment:
Still not aligned

Reply:
This is a github rendering issue because the code is using tabs.

Reply:
sorry, you're correct.